################################################################################
#_Parmfile_for_Migrate_5.0.4(git:distribution_version)_May_09_2022
#_DO_NOT_REMOVE_THE_FIRST_TWO_LINES!
#
#_generated_automatically_on
#_Thu_Apr__6_08:44:47_2023
#
#_please_report_problems_to_Peter_Beerli
#__email:_beerli@fsu.edu
#__http://popgen.sc.fsu.edu/migrate.html
################################################################################
#
################################################################################
#_General_options
################################################################################
#
#_Interactive_or_batch_job_usage
#___Syntax:_menu=<_YES_|_NO_>_
#_For_batch_runs_it_needs_to_be_set_to_NO
#_menu='YES'
#
#_Specification_of_length_of_names_of_individuals_(default=10)
#____Syntax:_nmlength=<INTEGER_between_0_.._30>
nmlength=12
#
#
#_If_you_long_runs_fails_because_of_timelimits
#_AND_you_have_specified_bayes_allfile='YES'....
#_then_you_can_recover_and_continue
#_by_setting_recover_to_yes,_default_is_NO
#_This_option_fails_if_your_bayesallfile_is_complete!
#_At_the_moment,_I_have_no_experience_about_failure_rate_etc._[March_30_2015]
#__________recover=<YES_|_NO>_
#
################################################################################
#_Data_options
################################################################################
#
#_Several_different_main_datatypes_are_possible:
#_INFINITE_ALLELE:_usable_for_electrophoretic_markers,
#__________________other_markers_with_unknown_mutation_model
#_STEPWISE_MUTATION:_usable_for_microsatellite_data_or
#__________________other_markers_with_stepwise_change
#__________________from_one_allele_to_another
#__________________[singlestep_versus_multistep_model,_see_micro_submodel_option]
#_FINITE_SITES_MUTATION:_standard_DNA/RNA_sequence_mutation
#__________________model,_usable_for_DNA_or_RNA_contiguous
#__________________sequences_or_variable_sites_only_(SNP)
#_______________________________________________________________________________
#_INFINITE_ALLELE
#__Syntax:_datatype=ALLELICDATA_
#__________#include_unknown=<YES_|_NO>_with_YES_unknown_alleles
#________________are_#included_into_analysis,_NO_is_the_default
#
#_______________________________________________________________________________
#
#_STEPWISE_MUTATION
#__Syntax:_datatype=<MICROSATELLITEDATA_|_BROWNIANDATA
#________________MICRO_specifies_the_standard_stepwise_mutation
#________________model,_the_BROWNIAN_is_an_approximation_to_this
#__________FOR_99%_of_all_cases_BROWNIAN_WILL_BE_FASTEST_AND_'BEST'
#__________for_MICRO_several_suboptions_are_available,_again_use_BROWNIAN!_
#__________micro_submodel=<1|2:{tune,pinc}>
#_________________1_means_singlestep_mutation_model_(this_is_the_default_and_the_standard
#_________________2_is_the_Multistep_model_(see_Watkins_2007_TPB,_section_4.2)_it_needs
#___________________two_parameters:_tune_specifies_how_close_the_model_is_to_a_singlestep_model
#___________________so_tune=0___>_singlestep,_tune=1___>_infinite_allele_model;
#___________________the_second_parameter_defines_the_probability_that_the_repeat_number
#___________________is_increasing,_this_value_cannot_be_larger_than_0.666,_I_suggest_0.5.
#___________________Example:_micro_submodel=2:{0.5,0.5}
#__________micro_threshold=<INTEGER>_Default_is_10_[MICRO_only,_NEEDS_TO_BE_EVEN!],
#________________smaller_values_speed_up_analysis,_but_might_also
#________________crash,_large_values_slow_down_analysis_considerably.
#________________Change_this_value_only_when_you_suspect_that_your
#________________data_has_huge_gaps_in_repeat_length.
#__________#include_unknown=<YES_|_NO>_with_YES_unknown_alleles
#________________are_#included_into_analysis,_NO_is_the_default
#
#_______________________________________________________________________________
#
#_FINITE_SITES_MUTATION
#__Syntax:_datatype=<SEQUENCEDATA_|_NUCLEOTIDE
#_________SEQENCEDATA:_typical_linked_stretches_of_DNA,_for_example_mtDNA
#_________NUCLEOTIDE:_linked_DNA_stretches,_all_invariable_sites_removed
#
#__________freqs_from_data=<YES_|_NO:_freq(A),_freq(C),_freq(G),_freq(T)>
#________________calculate_the_prior_base_frequencies_from_the_data,
#________________or_specify_the_frequencies
#__________ttratio=<RATIO1_RATIO2_....>_Default_is_2.0,
#________________ratio_between_transitions_and_transversions.
#__________seqerror_rate=<{VALUE,VALUE,VALUE,VALUE}|Estimate:1|4>_Default_is_0.0,_typical_values_for_ABI_3700_
#________________sequencers_after_base_calling_are_around_0.001_(1/650)
#__________categories=<VALUE:CATFILE>_The_categories_are_integers_or_letters
#________________specified_in_file_called_CATFILE,_this_assumes_that_all
#________________sites_belong_to_known_categories,_this_can_be_used_to
#________________weight_third_positions_etc.
#__________rates=<VALUE1_VALUE2_...>_the_rates_are_specified_arbitrarily_or
#________________then_are_from_a_Gamma_distribution_with_alpha=x,_currently
#_________________the_alpha_value_gets_lost_and_is_not_recorded_in_the_parmfile
#__________prob_rates=<RATE2_RATE1_..._>_These_rates_can_be_arbitrary_or_
#________________generated_with_gamma_deviated_rates_and_then_are_derived
#________________using_Laguerre's_quadrature,_this_should_get_better
#________________results_than_equal_probability_methods.
#__________autocorrelation=<NO_|_YES:VALUE>_Default_is_NO
#________________autocorrelation_makes_only_sense_with_rates,
#________________VALUE_should_be_>1.0
#__________weights=<NO_|_YES:WEIGHTFILE>_The_weights_are_specified
#________________in_file_called_WEIGHTFILE,_this_assumes_that_all_sites
#________________belong_to_known_weights,_this_can_be_used_to_weight
#________________portions_of_the_sequence_etc.
#__________fast_likelihood=<YES_|_NO>_Default_is_NO
#_______________________________________________________________________________
#__________population_relabel=={assignment_for_each_location_in_the_infile}
#________________example_is_population_relabel=={1_2_2}
#__________#random_subset=number<:seed>
#________________allows_to_subset_the_dataset_#randomly,_if_number_>_sample_in_population
#________________all_samples_are_taken,_if_number_is_smaller_then_the_pop_sample_is_shuffled_and
#________________and_the_first_number_samples_are_taken.
#________________the_#random_number_seed_guarantees_that_the
#________________same_subset_is_chosen_in_different_runs
datatype='BrownianMicrosatelliteData'
#include_unknown='NO'
datamodel='BM (BROWNIAN_MOTION)'
#
#__inheritance_scalars=<NO|YES:{values_for_each_locus}>
#________these_values_are_multiplied_with_Theta,_for_example_having
#________two_autosomal_and_a_locus_on_X__and_one_on_Y_chromosome_we_would_give_
#________inheritance_scalars='YES':{1_1_1.33_4.0}
#________#_this_will_scale_Theta_=4*N*mu
#________#_We_can_scale_differently:
#________inheritance_scalars='YES':{0.25_0.25_0.33_1.00}
#________#_this_will_scale_Theta_=N*mu_all_is_in_scale_for_Y_or_mtDNA
#________the_first_locus_is_the_reference_and_the_combined_estimate_has_its_inheritance_scalar
#________if_loci_are_{mtDNA,nucDNA}_then_{1.0_0.25}___>_Theta=N*mu_for_all
inheritance_scalars='NO'
#
#
#____Haplotyping:_
#____Syntax:_haplotyping=<YES:<report|no_report>_|_NO_>_
#____If_you_have_diplotype_data_input_two_lines_per_individual_marking
#____individual_names_name:other_where_name_must_be_on_both_lines,_it_will
#____will_be_used_to_build_the_indiviudal_database_for_the_haplotyping.
#____If_you_are_not_interested_in_the_haplotypes,_refrain_from_reprting_the
#____the_haplotypes_because_this_will_be_very_time_consuming_and_slows_the_program
haplotyping='NO'
#
#
#_______population_relabel=={assignment_for_each_location_in_the_infile}
#_____________example_is_population_relabel=={1_2_2}
population_relabel='{1,_2,_2,_2,_2,_2,_2,_2,_2,_2,_3,_3,_3}'
#
#__#random_subset=number:#random_numberseed
#________allows_to_subset_the_dataset_#randomly,_if_number_>_sample_in_population
#________all_samples_are_taken,_if_number_is_smaller_then_the_pop_sample_is_shuffled_and
#________and_the_first_number_samples_are_taken
#random_subset=10:3176820
#
################################################################################
#_Input_options
################################################################################
#
#_input_file_location
#___Syntax_infile=FILEPATH
infile='infile.msat.Myotis.myotis_13_pop'
#
#_#random_number_seed_specification
#___Syntax_#random_seed=<AUTO_|_OWN:<_seedfile_|_value_>
#______AUTO___________uses_computer_system_clock_to_generate_seed
#______OWN:seedfile___uses_file_seedfile_with_#random_number_seed
#______OWN:value______uses_number_value_for_seed
#random_seed=OWN:3176820
#
#_Specify_the_title_of_the_run,_will_be_overridden_by_title_in_datafile
#____Syntax:_title=title_text_[up_to_80_characters]
title='MICROSAT._Myotis_m._13_sampl.loc._3_pop._n_island'
#
#
################################################################################
#_Output_options
################################################################################
#
#_Progress_report_to_the_window_where_the_program_was_started
#____Syntax:_progress=<NO_|_YES>
#__________NO_______nothing_is_printed_to_the_console
#__________YES______some_messages_about_progress_are_reported_[default]
progress='YES'
#
#_______________________________________________________________________________
#
#_Recording_messages_to_screen_and_into_logfile
#___Syntax_logfile=<NO_|_YES:logfilename>
#_______NONE_____no_recording_of_progress
#_______logfilename__path_to_logfile
#___[this_is_usually_a_bad_choice!_Because_in_batch_jobs]
#____the_system_logs_the_screen,_on_macs_you_can_copy_the_screen,
#____only_on_windows_you_may_need_this;_this_slows_down_the_run_a_little
logfile='logfile_msat_13_pop_27_mag'
#
#_______________________________________________________________________________
#
#_Print_the_data_as_read_into_the_program
#___Syntax_print_data=<NO_|_YES>
#print_data='YES'
#
#_______________________________________________________________________________
#
#_Print_output_to_file_[default_is_outfile]
#___Syntax_outfile=outfilename
outfile='outfile_msat_13_pop_27_mag'
#
#_______________________________________________________________________________
#
#_Print_output_to_a_PDF_file_[default_is_outfile.pdf]
#___Syntax_pdf_outfile=outfilename.pdf
pdf_outfile='outfile_msat_13_pop_27_mag.pdf'
#
#_______________________________________________________________________________
#
#_Reduce_size_of_PDF____print/plot_only_summaries_[default_is_NO]
#___Syntax_pdf_terse=<NO|YES>
pdf_terse='NO'
#
#_______________________________________________________________________________
#
#_Report_M_(=migration_rate/mutation_rate)_instead_of_4Nm_or_2_Nm_or_Nm
#___Syntax_use_M=<NO_|_YES>_Default_is_YES,_the_name_4Nm_is_ambiguous
#______for_non_diploid_data
use_M='YES'
#
#_______________________________________________________________________________
#
#_Print_tree_into_treefile
#___Syntax_print_tree=<_NONE_|_<ALL_|_BEST_|_LASTCHAIN:Increment>:treefile_>
#_________NONE_no_tree_printed_[Default,_and_only_choice_using_parallel
#_________ALL__print_all_visited_genealogies_[careful_this_will_be_huge]
#_________BEST_print_only_the_best_tree_visited
#_________LASTCHAIN_print_all_trees_in_last_chain
#_________with_increment_INCREMENT
print_tree='NO'
#
#_______________________________________________________________________________
#
#_Print_a_histogram_of_the_time_of_migration_events_for_each_M(i_>j)
#____Syntax__mig_histogram=<NO_|_<ALL_|_MIGRATIONEVENTSONLY>:binsize:mighistfile_>
#_________NO____________do_not_record_any_events
#_________ALL___________record_migration_and_coalescence_event
#_________MIGRATIONEVENTSONLY_record_only_migration_events
#_________binsize_has_to_be_in_mutation_units,_with_an_average_Theta=0.01_try_0.001
#_Print_a_histogram_of_the_parameters_through_time_(skyline_plot)
#____Syntax__skyline=<NO_|_YES_|_PARAM:n_>:binsize:skylinefile_>
#_________NO____________do_not_calculate_parameter_estimates_through_time
#_________YES___________calculate_parameters_through_time
#_________binsize_has_to_be_in_mutation_units,_with_an_average_Theta=0.01_try_0.001
#_________If_the_interval_is_too_fine_the_output_will_be_very_noisy
mig_histogram='NO'
skyline='NO' #needs_mig_histogram=ALL:...
#
#
################################################################################
#_Parameter_start_settings
################################################################################
#
#___Syntax:_theta=<#random_|_OWN:<{value}_|_{value1,_value2,_....,_valuen}_|_PRIOR:percentage_>
#______migration==<#random_|_OWN:<{value}_|_{value1,_value2,_....,_valuen}_|_PRIOR:percentage_>
#________OWN_____starting_values_are_supplied_by_user
#___________{value}___if_only_one_value_is_supplied_then_all_population
#_____________________have_the_same_starting_value
#___________{value1,_value2,_...,_valuen}_each_population_has_its
#_____________________own_starting_value,_if_the_number_of_values_is
#_____________________insuffient,_then_the_last_value_is_the_template
#_____________________for_the_remaining_populations
#________#random__starting_parameter_is_drawn_#randomely_from_prior_distribution
#________PRIOR___starting_parameter_is_the_value_at_the_CDF(percentage)_of_the_prior_distribution
#___________percentage_is_a_value_between_0_and_100
theta='random'
migration='random'
rate='random'
split='random'
splitstd='random'
#
#_______________________________________________________________________________
#_Mutation_rate_modifiers
#
#___Syntax:_mutation=<CONSTANT_|_ESTIMATE_|_OWN:loci:_rate1_rate2_..._rate_loci>
#______CONSTANT_____all_loci_have_same_mutation_rate
#______ESTIMATE_____BAYESIAN_estimate:_mutation_rate_is_drawn_from_prior
#______OWN__________mutation_rate_is_different_for_every_locus,_but_fixed
#_________:loci:_rate1,_..._____number_of_loci,_rate_of_locus_1,_locus_2_etc.
#______DATA_________mutation_rate_modifier_is_deducted_from_loci_in_the_data
#___________________using_Watterson's_Theta_and_then_scaling_all_rates_Theta_locus/mean(Theta_loci
mutation='CONSTANT'
#
#
#_Treatment_of_invariant_sequence_loci
#_Syntax:_analyze_loci=<A_|_F_|_V>
#_________A_=analyze_all_loci_(Default!)
#_________F_=analyze_all_variable_loci_and_ONE_invariant_and_extrapolate
#_________V_=analyze_only_variable_loci
analyze_loci='A'
#
#_______________________________________________________________________________
#
#_Distribution_of_divergence_paramemter
#_Syntax:_divergence_distrib=<E_|_N>
#_________E_=Exponential_=use_exponential_distribution_(Default!)
#_________N_=Normal_=use_Normal_distribution
divergence_distrib='N'
#
#_______________________________________________________________________________
#
#_Custom_migration_model
#
#____Syntax:_custom_migration=={ab..bbab..ba_..._a}
#________the_{}_is_a_square_matrix_with_values_for_the_population_sizes
#________on_the_diagonal_and_migration_rates_or_divergences_off_diagonal
#________the_values__a__for_the_diagonal_can_be_any_of_these:
#________c_constant,_define_the_values_in_the_theta_option
#________*_[or_x]_free_to_vary,_the_default_is_*_for_every_parameter
#________m_mean_of_theta,_this_can_be_a_subgroup
#__________for_example:_theta_1_3_are_averaged,_thetas_4,5_are_estimated
#
#________the_values__b__for_the_migration/divergence_rates_can_be_any_of_these:
#________c_______constant,_the_value_needs_to_be_defined_in_the_migration_option
#________*_______migration_rate_free_to_vary,_this_is_the_default
#________d_______row_population_is_offspring_of_column_population
#________D_______row_population_is_offspring_of_column_population_with_migration
#
#________m_______mean_of_M_ij,_this_can_be_a_subgroup_of_migration_rates
#________________for_example_the_M_1_3i_are_averaged_and_M_4,5i_are_estimated
#________M_______means_of_4Nm_(diploid),_2Nm_(haploid),_Nm_(mtDNA,_Y_chromosome)
#________s_______symmetric_migration_rates_M
#________S_______symmetric_migrants_4Nm
#________a,b,c,d,...__any_parameter_group_label_for_migration_and_theta
#________________this_allows_using_averages_of_multiple_groups_[se_example]
#________
#________an_example_for_5_populations_could_look_like_this:
#________custom_migration=={*s00s
#__________________________s*s00
#__________________________0s*s0
#__________________________00s*s
#__________________________s00s*}
#________this_describes_a_circular_stepping_stone_model_with_5_symmetric_rates
#_________and_independent_sizes,_a_stepping_stone_with_3_parameters_would
#________look_like_this_custom_migration=={ma00b_bma00_0bma0_00bma_a00bm}
#________showing_two_different_migration_rates:_clockwise_and_counter_clockwise
#________population_splitting_example_with_3_populations:
#________custom_migration=={*D0_**d_00*}
#_____________________*_____1_____1_receives_migrants_from_2
#_____________________|MMMMM________and_split_from_2_
#__________________*________2_____2_receives_migrants_from_1
#__________________|________________and_split_from_3
#__________________*________3_____3_is_the_persisting_ancestor
custom_migration='{**}'
#
#
#_______print_the_parmfile_entry_for_the_population_growth_parameter_setting_and_labeling
#_______0='NO'_growth,_other_labels_are_marking_growth_either_individual_populations_or_combinations
#_______for_example:
#_______population_growth=={0}_#_all_populations_are_constant
#_______population_growth=={1}_#_all_populations_are_exponentially_growing_with_the_same_growth_rate
#_______population_growth=={1_2_3}_#_all_3_populations_are_growing_with_individual_rates,_if_there_are_more_than
#_________________________________#_than_3_populations,_pop_4_etc_will_grow_in_lockstep_with_3
#_______population_growth=={1_0_1}_#_population_1_and_3_grow_in_lockstep,_population_2_is_constant_in_size.
population_growth='{0}'
#
#_Influence_of_geography_on_migration_rate
#_a_distance_matrix_between_populations_changes_the_migration_rate_matrix_so_that
#_(genetic?)_migration_rates_=_inferred_migration_rate_/_distance_~_a_dispersion_coefficient
#_the_geofile_contains_a_number_of_populations,_names_for_populations_(10_characters),_they
#_need_to_be_in_order_of_the_dataset._And_the_distances_between_the_populations,_they_do_not
#_need_to_be_symmetric;_useful_distances_are_relative_to_1.0
#_for_example_using_km_with_large_distances_may_lead_to_very_large
#_values_since_they_are_per_distance_unit.
#____Syntax:_geo:<NO_|_YES:filename>
#_____________NO_______distances_among_populations_are_considered_to_be_1_[all_equal]
#_____________YES______distances_are_read_from_a_file
geo='NO'
#
#
################################################################################
#_Search_strategies
################################################################################
#
#_Bayesian_MCMC_Strategy_method
#_______updatefreq=VALUE_VALUE_VALUE_VALUE_VALUE_VALUE
#_________VALUE_is_a_ratio_between_0_and_1
#_______________ratio_of_how_many_times_the_genealogy_is_updated_compared_to_the_parameters
#_______________If_the_value_is_0.4_in_a_2_population_scenario_and_with_1000000_steps
#_______________The_tree_will_be_evaluated_400000_times,_Theta_1,_Theta_2,_M_21,_and_M_12
#_______________will_be_each_evaluated_125000_times._The_second_value_is_the_ratio_for_parameter
#_______________updates,_and_the_third_value_is_the_frequency_of_hapltype_updates.
#_______________the_fourth_values_is_for_assignment_of_individual_updates
#_______________The_values_do_not_need_add_up_to_1.0_but_will_be_recalculated_to_do_so
#_______________For_example:_1.0_2.0_0.1_0.0_results_in_0.32_treeupdates_0.65_parameter_
#_______________updates_and_0.03_haplotype_updates
#________bayes_posteriorbins=VALUE_VALUE
#____________VALUE______is_the_number_of_bins_in_the_posterior_distribution_histogram_for_Theta_or_M
#________bayes_posteriormaxtype=<_ALL_|_P99_|_MAXP99_|_TOTAL_>
#____________ALL________plots_the_WHOLE_prior_parameter_range
#____________P99________plots_from_the_minimum_prior_range_value_to
#_______________________the_99%_percentile_value_of_EACH_parameter
#____________MAXP99_____sets_all_axes_from_minimum_to_the_maximal
#_______________________99%_percentile_value_of_ALL_parameter
#____________TOTAL______plots_from_the_minimum_prior_range_value_to
#_______________________the_100%_percentile_value_of_EACH_parameter
#________bayes_file=<YES:FILENAME|NO>
#____________FILENAME_is_the_name_of_the_file_that_will_contain
#____________________the_results_for_the_posterior_distribution
#________bayes_allfile=<<YES|TEMP>:INTERVAL:FILENAME|NO>
#____________FILENAME_is_the_name_of_the_file_that_will_contain
#____________________all_parameters_of_the_posterior_distribution_[HUGE]
#____________INTERVAL_is_the_interval_at_which_all_parameters_are_written_to_file

#_____PROPOSAL:
#________bayes_proposals='THETA_<_SLICE_|_METROPOLIS_>
#________bayes_proposals='MIG_<_SLICE_|_METROPOLIS_>
#_______________SLICE_uses_the_slice_Sampler'_to_propose_new_parameter_values
#_______________METROPOLIS_uses_the_Metropolis_Hastings_Sampler'
#_______________(this_is_done_for_each_parameter_group:_THETA_or_MIGration)
#_____PRIORS:
#________Priors_can_be_set_for_each_parameter
#________There_are_several_ways_to_set:_a._old_format,_b._new_format_for_all,_c._individual
#________bayes_priors=FORCE_<PRIORdistribution_priorvalues>_#_a._all_in_FORCE
#________bayes_priors=FORCE_*_*_<PRIORdistribution_priorvalues>_#_b._all_in_FORCE
#________bayes_priors=FORCE_from_to_<PRIORdistribution_priorvalues>_#_c._individual,
#____________________________________from_and_to_are_population_numbers
#________FORCE_is_one_of_THETA,_MIG,_RATE,_SPLIT,_or_SPLITSTD
#_______________THETA_is_used_for_population_size_parameters
#_______________MIG_is_used_for_migration_rate_parameters
#_______________RATE_is_used_for_evolutionary_rate_differences_(use_only_with_date_samples)
#_______________SPLIT_is_used_for_mean_of_the_normal_distributed_population_divergence
#_______________SPLITSTD_is_used_for_the_standard_deviation_of_the_population_divergence
#________PRIORdistribution_is_one_of_UNIFORMPRIOR,_EXPPRIOR,_WEXPPRIOR,_GAMMAPRIOR,_NORMALPRIOR
#________________unipriorvalues:_min_max_delta
#________________exppriorvalues:_min_mean_max
#________________wexppriorvalues:_min_mean_max_delta
#________________gammapriorvalues:_min_mean_max_alpha
#________________normalpriorvalues:_min_mean_max_std
#
#_Search_OPTIONS
#________long_inc=VALUE______VALUE_is_the_number_of_updates_that_are_not_recorded
#________long_sample=VALUE___VALUE_is_the_number_of_sampled_updates
#________burn_in=VALUE_______VALUE_is_the_number_of_updates_to_discard_at_the_beginning
#________auto_tune=<NO_|_YES:VALUE>__VALUE_the_the_target_acceptance_ratio
#_____________________________if_value_is_missing,_it_is_set_to_0.44
#________assign=<YES:<FREQ|UNIFORM<:{pop1,pop2,...}>>_|_NO>____YES_will_assign_individuals_to_populations
#_____________________________with_'?'_as_first_character
#_____________________________[FREQ_uses_past_assignment_during_the_MCMC
#_____________________________UNFORM_picks_a_#random_population
#_____________________________optional_list_if_populations_to_assign_to
#_____________________________can_be_supplied_using_:{pop1,pop2,...}
#
#_updatefreq=tree_parameter_haplotype_timeparam_assignment_seqerror,_ml_alpha
#updatefreq=0.200000_0.200000_0.200000_0.200000_0.200000_0.100000_0.000000_
#bayes_posteriorbins=1500_1500_1500_1500_1500_
bayes_posteriormaxtype='TOTAL'
bayes_file='bayesfile_msat_13_pop_27_mag'
bayes_allfile='NO'
bayes_all_posteriors='NO'
bayes_proposals='THETA_SLICE_Sampler'
bayes_proposals='MIG_SLICE_Sampler'
bayes_proposals='DIVERGENCE_METROPOLIS_HASTINGS_Sampler'
bayes_proposals='DIVERGENCESTD_METROPOLIS_HASTINGS_Sampler'
bayes_proposals='GROWTH_METROPOLIS_HASTINGS_Sampler'
#bayes_priors=THETA_*_*_UNIFORMPRIOR:_0.000000_1.00000_0.10000_
#bayes_priors=MIG_*_*_UNIFORMPRIOR:_0.000000_1000.000000_100.000000_
#bayes_priors=RATE_*_*_UNIFORMPRIOR:_0.000000_10000000000.000000_1000000000.000000_
#
#__Hyper_prior_for_all_parameters
#__The_parameter_of_the_prior_is_drawn_from_a_Gamma_distribution_with_mean_and_alpha
#__for_example:
#____bayes_hyperprior='YES':10000:1.0:5.0
#__uses_a_hyper_prior_with_the_mean_of_the_specified_prior
#__and_and_alpha_so_that_this_specifies_~Normal
#
bayes_hyperpriors='NO'
#
long_chains=1
long_inc=100
long_sample=10000
burn_in=100
#auto_tune='YES':0.440000
assign='NO'
#
#_______________________________________________________________________________
#_Schemes_to_improve_MCMC_searching_and/or_thermodynamic_integration
#
#_Heating_schemes_{MCMCMC_=MC_cubed}
#____Syntax:_heating=<_NO_|_<YES_|_ADAPTIVE>:SKIP:TEMPERATURES
#________NO____No_heating
#________YES___heating_using_TEMPERATURES
#________ADAPTIVE_adaptive_heating_using_start_TEMPERATURES_[fails_sometimes!!]
#________SKIP_skip_that_many_comparisons,_this_lengthens_the_run_by_SKIP
#____________TEMPERATURES____{_1.0,_temp1,_temp2,_temp3_.._tempn}
#_____Example:_heating='YES':1:{1.0,_1.2,_3.0,1000000.0}
#_Heating:__swapping_chains
#_____Syntax:_heated_swap=<_YES_|_NO_>
#_________YES__swapping_of_chains_enabled_[DEFAULT]
#_________NO___swapping_of_chains_disabled
#______Example:_heated_swap='YES'
heating='YES:1:{1.0,1.071,1.154,1.25,1.364,1.5,1.667,1.875,2.143,2.5,3.0,3.75,5.0,7.5,15.0,1000000.0}'
heated_swap='YES'
#
#_Lengthening_chain_schemes
#____Syntax:_moving_steps=<_NO_|_YES:VALUE>
#_______VALUE___frequency_is_between_0..1
moving_steps='NO'
#
#
#____Convergence_statistic_[Gelman_and_Rubin]
#____Syntax:_gelman_convergence=<_YES:Pairs|Summary_|_NO_>
#_______NO______do_not_use_Gelman's_convergence_criterium
#_______YES_____use_Gelman's_convergence_criteria_between_chain_i,_and_i_1
#_______________PAIRS_reports_all_replicate_pairs
#_______________SUM___reports_only_mean_and_maxima
gelman_convergence='NO'
#
#____REPLICATON:
#____Syntax:_replicate=<_NO_|_YES:<VALUE>_>
#_______NO_____no_replication_of_run
#_______YES____replicate_run
#___________VALUE_____number_between_2_and_many,_complete_replicates
replicate='YES:10'
#
#_______________________________________________________________________________
#
#end
